package.json

```diff
@@ -21,7 +21,7 @@
   ],
   "dependencies": {
     "event-kit": "^2.0.0",
-    "nan": "^2.0.0"
+    "nan": ">2.7.0"
```

Comment:
I think this version range is unnecessarily broad and might lead to trouble down the road.  Could you change the `>` back to `^`?  Once that's done I'll be happy to merge this.

Reply:
That will just result in this very same issue, builds breaking with new nodejs versions. This may not be a big deal to you but for packagers like me, it is. When node updates again and breaks nan, we'll have to wait for *every* nested dependency of our app to add a new releases. If yarn would let us overwrite nested dependencies this wouldn't be a problem but unfortunately we can't do that. Thank you.

Reply:
Does `^2.0.0` not give you `2.10.0` in practice?  `nan` hasn't bumped major versions in 3 years so this version range would keep you up to date in the 2.x series.  When I run `npm install` in `master` of this repo, I get this dependency tree:

```
$ npm ls
keyboard-layout@2.0.13 /home/daviwil/Projects/Code/keyboard-layout
├── event-kit@2.5.0
├─┬ jasmine-focused@1.0.7
│ ├─┬ jasmine-node@1.10.2 (git+https://github.com/kevinsawicki/jasmine-node.git#81af4f953a2b7dfb5bde8331c05362a4b464c5ef)
│ │ ├── coffee-script@1.12.7
│ │ ├─┬ coffeestack@1.1.2
│ │ │ ├─┬ coffee-script@1.8.0
│ │ │ │ └── mkdirp@0.3.5 deduped
│ │ │ ├─┬ fs-plus@2.10.1
│ │ │ │ ├── async@1.5.2
│ │ │ │ ├─┬ mkdirp@0.5.1
│ │ │ │ │ └── minimist@0.0.8 deduped
│ │ │ │ ├─┬ rimraf@2.6.2
│ │ │ │ │ └─┬ glob@7.1.2
│ │ │ │ │   ├── fs.realpath@1.0.0
│ │ │ │ │   ├─┬ inflight@1.0.6
│ │ │ │ │   │ ├── once@1.4.0 deduped
│ │ │ │ │   │ └── wrappy@1.0.2
│ │ │ │ │   ├── inherits@2.0.3 deduped
│ │ │ │ │   ├─┬ minimatch@3.0.4
│ │ │ │ │   │ └─┬ brace-expansion@1.1.11
│ │ │ │ │   │   ├── balanced-match@1.0.0
│ │ │ │ │   │   └── concat-map@0.0.1
│ │ │ │ │   ├─┬ once@1.4.0
│ │ │ │ │   │ └── wrappy@1.0.2 deduped
│ │ │ │ │   └── path-is-absolute@1.0.1
│ │ │ │ └── underscore-plus@1.6.8 deduped
│ │ │ └─┬ source-map@0.1.43
│ │ │   └── amdefine@1.0.1
│ │ ├─┬ gaze@0.3.4
│ │ │ ├─┬ fileset@0.1.8
│ │ │ │ ├─┬ glob@3.2.11
│ │ │ │ │ ├── inherits@2.0.3
│ │ │ │ │ └─┬ minimatch@0.3.0
│ │ │ │ │   ├── lru-cache@2.7.3 deduped
│ │ │ │ │   └── sigmund@1.0.1 deduped
│ │ │ │ └─┬ minimatch@0.4.0
│ │ │ │   ├── lru-cache@2.7.3 deduped
│ │ │ │   └── sigmund@1.0.1 deduped
│ │ │ └─┬ minimatch@0.2.14
│ │ │   ├── lru-cache@2.7.3
│ │ │   └── sigmund@1.0.1
│ │ ├─┬ jasmine-reporters@2.3.2
│ │ │ ├─┬ mkdirp@0.5.1
│ │ │ │ └── minimist@0.0.8
│ │ │ └── xmldom@0.1.27
│ │ ├── mkdirp@0.3.5
│ │ ├── requirejs@2.3.5
│ │ ├── underscore@1.9.1
│ │ └── walkdir@0.0.7 deduped
│ ├─┬ underscore-plus@1.6.8
│ │ └── underscore@1.8.3
│ └── walkdir@0.0.7
└── nan@2.10.0
```

Notice the `nan@2.10.0` at the bottom.  If you're not getting this, perhaps npm or yarn is using some other logic to resolve `nan` to a lower version?

Reply:
Hum, I misread the error message. `keyboard-layout` is the one failing. I feel very stupid.

`npm ls`: http://ix.io/1kRY
build: http://ix.io/1kS3